### PR TITLE
Added basic Folder model with multiple folder structure

### PR DESCRIPTION
**Pull request by ayu023ban**
### Description

Created updated folder modal
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires migration.
- [ ] My change requires `npm install`.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -21,19 +22,61 @@ class Folder(Model):
     """
     This model holds information about a folder owned by a person
     """
+    
+
 
     person = models.OneToOneField(
```
Comment (Ajayneethikannan) [inline models.py]: person should be a foreign key

**Comment by Ajayneethikannan** [comment]
@ayu023ban i feel it would be better to add all the necessary models before merging this. Continue to add your commits here.

**Comment by ayu023ban** [comment]
we were mereging it in filemanager_revamp branch so it would not be reflected in default

File: models.py (hunk 1 above)
Comment (ayu023ban) [reply models.py]: will update it

**Pull request merged**